agir/front/components/SearchPage/SearchPage.js

```diff
@@ -470,7 +440,9 @@ export const SearchPage = () => {
                     placeholder="Date"
                     name="eventSort"
                     value={eventSort}
-                    onChange={handleChangeEventSort}
+                    onChange={(e) => {
+                      setEventSort(e);
+                    }}
```

Comment:
Ça suffit comme ça, non ?
```suggestion
                    onChange={setEventSort}
```

Reply:
En effet ! :)